### PR TITLE
MWPW-140836: fix Commerce for several geo's

### PR DESCRIPTION
**Pull request by 3ch023**
add a fix from Commerce with additional mapping for CIS and SEA
- https://git.corp.adobe.com/wcms/tacocat.js/pull/499 
- https://git.corp.adobe.com/wcms/tacocat.js/pull/497

Resolves: [MWPW-140836](https://jira.corp.adobe.com/browse/MWPW-140836)

merch should be working now for these geo's:
```
const GeoMap = {
    africa: 'en-ZA',
    mena_en: 'en-DZ',
    il_he: 'iw-IL',
    mena_ar: 'ar-DZ',
    id_id: 'in-ID',
    no: 'nb-NO',
    cis_en: 'en-AZ',
    cis_ru: 'ru-AZ',
    sea: 'en-SG',
    la: 'es-DO',
};
```

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/africa/drafts/mariia/commerce?martech=off
- After: https://MWPW-140836-wcsLocale--milo--adobecom.hlx.page/africa/drafts/mariia/commerce?martech=off

### CODE REVIEW

**Review by VKniaz — APPROVED** [review]

**Review by Axelcureno — APPROVED** [review]

File: libs/scripts/scripts.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const prodDomains = ['milo.adobe.com'];
 const locales = {
```
Comment (mokimo) [inline libs/scripts/scripts.js:22]: Be aware that each consumer defines their own locales, those are just the locales for milo.adobe.com.

Comment (mokimo) [reply libs/scripts/scripts.js:22]: Here a bacom exam

Comment (3ch023) [reply libs/scripts/scripts.js:22]: Yes, once we merge this PR I will reach out to consumers to review their configs.
My initial ticket was actually for CC, but I want Milo project to have a valid configuration and serve as a blueprint for consumer teams, thus fixing it here. 
Except for a few specific cases, most of the locales config should be the same across clouds.

**Comment by yesil** [comment]
@3ch023 can you link the source of truth you used for the new values?

**Comment by 3ch023** [comment]
> @3ch023 can you link the source of truth you used for the new values?

mostly CC Dexter Production, in different geos: [pricing-info.html](https://www.adobe.com/products/illustrator/pricing-info.html)

**Comment by 3ch023** [comment]
I split this PR in two.
This PR will brings the Commerce mapping only.
The second PR will be fixing the ietf values, since it has more impact and will need more discussions.

**Comment by 3ch023** [comment]
follow-up PR: [https://github.com/adobecom/milo/pull/1711](https://github.com/adobecom/milo/pull/1711)

**Review by yesil — APPROVED** [review]
This should have stayed in Milo at the first place IMO.
cc: @vladen

**Review by mokimo — APPROVED** [review]

**Pull request merged**